libethereum/BlockChainSync.cpp

```diff
@@ -935,7 +928,7 @@ void BlockChainSync::onPeerNewHashes(std::shared_ptr<EthereumPeer> _peer, std::v
     if (unknowns > 0)
     {
         LOG(m_loggerDetail) << "Not syncing and new block hash discovered: syncing.";
-        syncPeer(_peer, true);
+        syncPeer(_peerID, true);
```

Comment:
(finished here)

Reply:
Unfortunately we don't have any convention about capitalizing abbreviations in the names, so both ways can be found in the code